pkg/utils/utils.go

```diff
@@ -162,6 +162,17 @@ func AddJitter(duration time.Duration, maxJitterPercentage int) time.Duration {
 	return duration * time.Duration(jitter)
 }
 
+// Jitter returns a random duration
+func Jitter(duration *time.Duration, maxJitterPercentage float64) {
```

Comment:
I don't see why we have so many ways of dealing with durations... AddJitter, Jitter, RandomDuration, ...

Reply:
The other functions contains bugs :(